index.html

```diff
@@ -143,6 +143,31 @@ <h2 class="text-center text-bold text-lucida text-subheading mb-5">Testimonials<
             </div>
         </div>
     </section>
+    <footer id="footer" class="container">
+        <div class="d-flex flex-wrap pt-5 pb-5">
+            <a href="#" class="mr-3">
+              <img src="images/angellist.svg" alt="Icon Angellist" class="img-fluid" />
+            </a>
+            <a href="#" class="mr-3">
+              <img src="images/linkedin.svg" alt="Icon Linkedin" class="img-fluid" />
+            </a>
+            <a href="#" class="mr-3">
+              <img src="images/mail.svg" alt="Icon Envelope" class="img-fluid" />
+            </a>
+          </div>
+          <div class="background-blue"></div>
+          <div class="pt-3 pb-3 d-flex flex-wrap">
+            <div class="mr-auto pr-3">
+              Copyright &copy; 2018
+            </div>
+            <div class="d-flex align-items-center">
+              <span>Designed with</span>
+              <img class="ml-2 mr-2" src="images/heart-icon.svg" alt="Heart Icon" />
+              <span>by &nbsp;</span>
+              <a href="#" class="link-orange">Vlada Boiko</a>
```

Comment:
this is no mistake but the correct link is `https://www.behance.net/vladislava940a` :)

Reply:
Well played! 👍 )